scripts/postinstall.js

```diff
@@ -2,11 +2,12 @@ const fs = require('fs');
 const crypto = require('crypto');
 const fname = 'secrets.env';
 
-// two different clients need to
-// use the same api key, they
+// three different clients need to
+// use the same api key when bootstrapping
+// a docker environment, they
 // use different env variables
 // to read the value so
-// write the randomValue to two
+// write the randomValue to three
```

Comment:
Maybe we should just get rid of the specific number so we don't have to update it anymore